client/src/app/pages/forum/forum.component.ts

```diff
@@ -1,12 +1,21 @@
 import { Component } from '@angular/core';
+import { PostComponent } from '../../components/post/post.component';
+import { UserStoreService } from '../../services/user-store.service'
 
 @Component({
   selector: 'app-forum',
   standalone: true,
-  imports: [],
+  imports: [PostComponent, ],
   templateUrl: './forum.component.html',
   styleUrl: './forum.component.css'
 })
 export class ForumComponent {
+  constructor(
+    private userStore: UserStoreService
```

Comment:
Make this public so you can access the `isAuthenticated()` method directly in the template